### PR TITLE
Api to show details about the conference

### PR DESCRIPTION
**Pull request by narayanaditya95**
@gvwilson @aterrel @martey 

For PyData Seattle, `api/conference` returns

```json
{
    "title": "PyData Seattle 2015",
    "start_date": "2015-07-24",
    "end_date": "2015-07-26"
}
```

This should be merged after #115

### CODE REVIEW

**Comment by gvwilson** [comment]
Why rename `name` to `personal` ?  The latter usually means "personal 
name", like "Greg" (as opposed to family name, like "Wilson").

**Comment by narayanaditya95** [comment]
@all please review the latest commit. The previous two commit belong to #115.

**Comment by narayanaditya95** [comment]
Rebased against #115. The details of the conference is available at `/api/`.

**Comment by narayanaditya95** [comment]
Added a viewset to list and show details regarding the `Presentation` model.
A sample response would look like:
```
{
    "speaker": {
        "username": "admin",
        "name": "admin",
        "email": "admin@pydata.org"
    },
    "slot": {
        "day": "2016-06-14",
        "kind": "Test",
        "name": "2016-06-14 Test (09:55:44 - 00:00:00) ",
        "start": "09:55:44",
        "end": null
    },
    "section": "PyData General Sessions",
    "title": "Big Data",
    "description": "Test",
    "abstract": "Test",
    "cancelled": true,
    "additional_speakers": []
}
```
@martey Please review.

**Comment by narayanaditya95** [comment]
@martey ping.

File: conf_site/api/views.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-from rest_framework import response
+from rest_framework import views
+from rest_framework.response import Response
 from rest_framework.permissions import IsAdminUser
+from symposion.conference.models import Conference
 from symposion.speakers.models import Speaker
+from symposion.schedule.models import Presentation
 
-from .serializers import SpeakerSerializer
+from .serializers import *
```
Comment (martey) [inline conf_site/api/views.py]: Can we avoid using a wildcard import here? It's generally considered a Python antipattern (and is warned against in [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)).

File: conf_site/api/views.py [hunk 2]
```diff
@@ -18,3 +21,25 @@ class SpeakerViewSet(viewsets.ReadOnlyModelViewSet):
...
+
+    def get(request, *args, **kwargs):
+        conference = Conference.objects.first()
+        serializer = ConferenceSerializer(conference)
+        return Response(serializer.data)
+
+
+class PresentationViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Returns a the list of all sections within the conference.
```
Comment (martey) [inline conf_site/api/views.py]: I think this docstring might be mistaken - I assume `PresentationViewSet` returns a list of presentations in the conference, not sections.

**Comment by narayanaditya95** [comment]
@martey I addressed the issues you raised.

**Comment by narayanaditya95** [comment]
Closing this PR as it is contained in #121

**Pull request closed without merging**